workspace/welcome-to-xod/004-patching/patch.xodp

```diff
@@ -1,31 +1,45 @@
 {
   "comments": [
     {
-      "content": "# Patching\n\nNow it's time to learn how to add nodes and link them together. Let's recreate our counting program from scratch so we'll have two independent counters.\n\n## Instructions\n\n1. *Double-click* on some empty space on the patch or press the \"i\" key on your keyboard.\n\n2. The *Suggester* will appear. Type `clock` into it and press *enter*\n\n3. A new `clock` node will appear in the place you double-clicked in step 1.\n\n4. Try moving a node by clicking and dragging it.\n\n5. Repeat steps 2-4 to add `count` and `watch` nodes.\n\n6. Create a link between `clock`'s `TICK` pin and `count`'s `INC` pin by clicking on one pin, then on the other.\n\n7. Create a link between `count`'s output pin and `watch`'s input pin.\n",
+      "content": "## 👆 Ways to add node\n\nAlternatively to double-clicking you can also press the “I” key or hit Edit → Add Node.\n\nIf you don’t remember the name of a node, try to find it in Project Browser. Once you spot it there, drag’n’drop the item to the patch board or hit Place in its context menu.",
+      "id": "BJJH5p-cV",
+      "position": {
+        "units": "slots",
+        "x": -2,
+        "y": 6
+      },
+      "size": {
+        "height": 2,
+        "units": "slots",
+        "width": 8
+      }
+    },
+    {
+      "content": "# Patching\n\nYou guess correctly, you may add nodes and link them to build brand new programs and improve existing.\n\n## Exercise\n\nLet's extend our counting program so that we'll have two independent counters.\n\n1. Double-click on some empty space on the patch; the suggester line will appear.\n2. Start typing “clock” into it and press Enter once you see `xod/core/clock`.\n3. A new `clock` node will appear where you have double-clicked.\n4. Move the node around by dragging it.\n5. Repeat steps 1-4 to add `count` and `watch` nodes.\n6. Create a link between `clock`'s `TICK` pin and `count`'s `INC` pin by clicking on one pin, then the other.\n7. Create a link between `count`'s output pin and `watch`'s input pin.\n\nRun the simulation to see how the counters work in parallel. Try to set different `IVAL` parameters and see how it changes the behavior.",
```

Comment:
```suggestion
      "content": "# Patching\n\nYou guessed correctly, you may add nodes and link them to build brand new programs and improve existing ones.\n\n## Exercise\n\nLet's extend our counting program so that we'll have two independent counters.\n\n1. Double-click on some empty space on the patch; the suggester line will appear.\n2. Start typing “clock” into it and press Enter once you see `xod/core/clock`.\n3. A new `clock` node will appear where you have double-clicked.\n4. Move the node around by dragging it.\n5. Repeat steps 1-4 to add `count` and `watch` nodes.\n6. Create a link between `clock`'s `TICK` pin and `count`'s `INC` pin by clicking on one pin, then the other.\n7. Create a link between `count`'s output pin and `watch`'s input pin.\n\nRun the simulation to see how the counters work in parallel. Try to set different `IVAL` parameters and see how it changes the behavior.",
```